backend/sse/src/server.ts

```
@@ -9,42 +9,66 @@
+});
+
+redis.on("error", (err) => {
+    console.error(err);
+});
+
+
+// Keep track of connected clients
 const clients: Map<string, Response> = new Map();
 
+/****************** Implement streaming services ******************/
 const streamHiddenTestProcess = async (call: any, callback: Function) => {
```

Biome (1.9.4)

[error] Don't use 'Function' as a type.

Prefer explicitly define the function shape. This type accepts any function-like value, which can be a common source of bugs.

(lint/complexity/noBannedTypes)